docs/website/optimizations.md

```diff
@@ -769,3 +769,21 @@ a product assembly and does not automagically mark all `NSObject`
 subclasses. This allows additional removes and optimizations to be 
 applied to the assembly, including the ability to remove `UIWebView` if
 nothing else in the application requires it.
+
+## Force Rejected Types Removal
+
+This optimization can be enabled when it's not possible to use the 
+managed linker (e.g. **Don't link**) or when the managed linker cannot
+remove references to deprecated types that would cause an application
+to be rejected by Apple.
+
+References to the existing types will be renamed, e.g. `UIWebView` to 
+`DeprecatedWebView`, in every assemblies.
+
+The type definition is also renamed (for validity) and all custom 
+attributes on the types and their members will be removed. 
+Code inside the members will be replaced with a
+`throw new NotSupportedException ();`.
+
+The default behavior can be overridden by passing
+`--optimize=[+|-]force-rejected-types-removal` to `mtouch`.
```

Comment:
We should put a list of the removed types here or a link to where it lives in our source code

Reply:
a link is a good idea - since it would be too easy to get out of sync